pyrightconfig.testcases.json

```diff
@@ -2,50 +2,12 @@
     "$schema": "https://raw.githubusercontent.com/microsoft/pyright/main/packages/vscode-pyright/schemas/pyrightconfig.schema.json",
     "typeshedPath": ".",
     "include": [
-        "test_cases"
+        "test_cases",
     ],
-    "typeCheckingMode": "basic",
-    "strictListInference": true,
-    "strictDictionaryInference": true,
-    "strictParameterNoneValue": true,
-    "reportFunctionMemberAccess": "error",
-    "reportMissingModuleSource": "none",
-    "reportMissingTypeStubs": "error",
-    "reportUnusedImport": "error",
-    "reportUnusedClass": "error",
-    "reportUnusedFunction": "error",
-    "reportUnusedVariable": "error",
-    "reportDuplicateImport": "error",
-    "reportOptionalSubscript": "error",
-    "reportOptionalMemberAccess": "error",
-    "reportOptionalCall": "error",
-    "reportOptionalIterable": "error",
-    "reportOptionalContextManager": "error",
-    "reportOptionalOperand": "error",
-    "reportUntypedFunctionDecorator": "error",
-    "reportUntypedClassDecorator": "error",
-    "reportUntypedBaseClass": "error",
-    "reportUntypedNamedTuple": "error",
-    "reportPrivateUsage": "none",
-    "reportConstantRedefinition": "error",
-    "reportInvalidStringEscapeSequence": "error",
-    "reportUnknownParameterType": "error",
-    "reportUnknownArgumentType": "error",
-    "reportUnknownLambdaType": "error",
-    "reportUnknownVariableType": "error",
-    "reportUnknownMemberType": "error",
-    "reportMissingTypeArgument": "error",
-    "reportUndefinedVariable": "error",
-    "reportUnboundVariable": "error",
-    "reportInvalidStubStatement": "error",
-    "reportInvalidTypeVarUse": "error",
+    "typeCheckingMode": "strict",
     "reportPropertyTypeMismatch": "error",
-    "reportSelfClsParameterName": "error",
```

Comment:
Oh, I think it might be good to set this one to `"none"`, like we do with our other two configs, actually. It's not really that relevant for typeshed's purposes :)